svg.js.d.ts

```diff
@@ -429,8 +434,14 @@ declare namespace svgjs {
         f: number;
         matrix: Matrix;
     }
+
+    interface Library {
+        Matrix: Matrix;
+    }
+
     export interface Matrix {
         (source: any): Matrix;
+        new(source: any): Matrix;

```

Comment:
I had a lot of free time on my hands so i decided to fix this bug.
If you want to check it out, its over [here](https://github.com/rdfriedl/svg.js/commits/master)

Im not an expert in using git or github so im not sure how we would merge my code into this PR.
any ideas?